api/include/opentelemetry/trace/tracer.h

```diff
@@ -52,6 +52,13 @@ class Tracer
     return this->StartSpan(name, attributes, {}, options);
   }
 
+  nostd::shared_ptr<Span> StartSpan(nostd::string_view name,
```

Comment:
I think you're still missing methods that allows to pass custom non-standard `KeyValueIterable` objects together with links?

Reply:
Logged an issue here: https://github.com/open-telemetry/opentelemetry-cpp/issues/649